Ratings.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components'
+
+const changeColor = () => {
+    var colors = [`#21CE99`, `#F45531`];
+    return colors[Math.round(Math.random() * (1 - 0) + 0)]
+}
+
+const color = changeColor();
+
+const changeCircleColor = () => {
+    if (color === `#21CE99`) {
+        return `rgb(33, 206, 153, .3)`
+    } else {
+        return `rgb(244, 85, 49, .3)`
+    }
+}
+
+const circleColor = changeCircleColor();
+
+const Wrapper = styled.div`
+padding: 4em;
+background: #1B1B1D;
+display: grid;
+`;
+
+const RatingsTitle = styled.div`
+float: left;
+color: #ffffff;
+font-family: 'DINPro-Medium';
+font-size: 24px;
+`;
+
+const LineBreak = styled.hr`
+display: block;
+height: 1px;
+border: 0;
+border-top: 1px solid black;
+margin: 1em 0;
+margin-bottom: 25px;
+padding: 0;
+`;
+
+const MainContainer = styled.div`
+display: table;
+background-color: #1B1B1D;
+width: 100%;
+height: 200px;
+`;
+
+const RatingCircle = styled.div`
+  margin-left: 60px;
+  display: table;
+  float: left;
+  height: 130px;
+  width: 130px;
+  background-color: ${circleColor};
+  border-radius: 50%;  
+`;
+
+const CircleContent = styled.div`
+display: table-cell; 
+vertical-align: middle; 
+text-align: center; 
+font-family: 'DINPro-Medium';
+color: ${color};
```

Comment:
Where are you getting the ${color} from? Is it a a global styled component thing? I'm trying to do something similar